sentry-android-core/src/main/java/io/sentry/android/core/internal/modules/AssetsModulesLoader.java

```diff
@@ -0,0 +1,44 @@
+package io.sentry.android.core.internal.modules;
+
+import android.content.Context;
+import io.sentry.ILogger;
+import io.sentry.SentryLevel;
+import io.sentry.internal.modules.ModulesLoader;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.InputStream;
+import java.lang.ref.WeakReference;
+import java.util.Map;
+import java.util.TreeMap;
+import org.jetbrains.annotations.ApiStatus;
+import org.jetbrains.annotations.NotNull;
+
+@ApiStatus.Internal
+public final class AssetsModulesLoader extends ModulesLoader {
+
+  private final @NotNull WeakReference<Context> contextRef;
```

Comment:
`l`: is there a reason why you're using a `WeakReference` here? As far as I can see `AndroidOptionsInitializer` ensures that context is an application context already.

Reply:
No real reason, just extra-safety I guess 🤣